data-raw/ictr_data.R

```diff
@@ -1,12 +1,12 @@
+library(vroom)
 library(here)
-library(readr)
 library(usethis)
 
 # The raw data comes from tiltIndicator#87, which includes pre-processing
-ictr_inputs <- read_csv(here("data-raw", "ictr", "ictr_inputs.csv"), show_col_types = FALSE) |>
+ictr_inputs <- vroom(here("data-raw", "ictr", "real_data", "ictr_inputs.csv"), show_col_types = FALSE) |>
   select(all_of(ictr_inputs_crucial()))
 use_data(ictr_inputs, overwrite = TRUE)
 
-ictr_companies <- read_csv(here("data-raw", "ictr", "ictr_companies.csv"), show_col_types = FALSE) |>
+ictr_companies <- vroom(here("data-raw", "ictr", "real_data", "ictr_companies.csv"), show_col_types = FALSE) |>
```

Comment:
Why do we change to vroom?

Reply:
@maurolepore I experienced delimiting issues while reading the new csv file using `read_csv`, hence I used vroom. I tried many other options with `read_csv` for reading the new csv file but none worked. You can check from your end as well if `read_csv` works for you if you change the column names of existing csv file. If it will work for you then we can have a meeting to discuss my case and decide what to use.

Reply:
I can check later today. I suspect the issue relates to [this new guideline](https://gist.github.com/maurolepore/17b468dd8c3a603c668220563754ad7f#in-csv-files-use-the-global-format).

Reply:
@maurolepore I have read the guideline and tried different .csv formats to save the changes but it didn't work out. Please let me know if you want to show me the solution in a meeting! Thanks!

Reply:
@maurolepore  I tried to follow the guideline but it didn't work out due to an error. Please check from your side and let me know what worked for you! Thanks!

Reply:
Did  you try `read_csv2()`?

I see that `vroom()` guesses the correct delimiter but fails to understand the correct decimal point.

``` r
path <- here::here("data-raw", "ictr", "ictr_inputs.csv")

# Good
readr::read_csv2(path, col_select = "input_co2", , col_types = "d")
#> ℹ Using "','" as decimal and "'.'" as grouping mark. Use `read_delim()` for more control.
#> # A tibble: 75 × 1
#>    input_co2
#>        <dbl>
#>  1      73.6
#>  2      96  
#>  3      39.1
#>  4      80.1
#>  5      10.7
#>  6      25.7
#>  7      16.9
#>  8      55.8
#>  9      29.1
#> 10      92.7
#> # ℹ 65 more rows

# Bad: Uses the correct delimiter but the wrong decimal point
vroom::vroom(path, col_select = "input_co2", col_types = "d")
#> # A tibble: 75 × 1
#>    input_co2
#>        <dbl>
#>  1      7358
#>  2        96
#>  3      3907
#>  4      8012
#>  5      1072
#>  6      2571
#>  7      1693
#>  8      5582
#>  9      2913
#> 10      9274
#> # ℹ 65 more rows
```

Can you read the data with `read_csv2()` then rewrite it as the new raw data? Then read it with `read_csv()`.

Try one more time else I'll take over.